spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,6 +323,38 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response!(app_id: nil, platform: "ios", review_id: nil, response: nil)
+      raise "app_id is required" unless app_id
+      raise "review_id is required" unless review_id
+      raise "response is required" unless response
```

Comment:
thats a good point 👍  - not sure if i should mix this into this PR?
as @KrauseFx mentioned that there might be concerns about supporting dev response posting in general.

i think we should decide that first.